src/transformers/models/timesformer/configuration_timesformer.py

```diff
@@ -0,0 +1,128 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" TimeSformer model configuration"""
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+
+
+logger = logging.get_logger(__name__)
+
+TIMESFORMER_PRETRAINED_CONFIG_ARCHIVE_MAP = {
+    "facebook/timesformer": "https://huggingface.co/facebook/timesformer/resolve/main/config.json",
+}
+
+
+class TimeSformerConfig(PretrainedConfig):
+    r"""
+    This is the configuration class to store the configuration of a [`TimeSformerModel`]. It is used to instantiate a
```

Comment:
```suggestion
    This is the configuration class to store the configuration of a [`TimesformerModel`]. It is used to instantiate a
```
You can change the name of the classes everywhere to just lowercase `TimesformerModel` rather than `TimeSformerModel` => will be easier for people to type

Reply:
Yes it would be better with lower case `s` 👍 I have updated all class names.